ts/session/utils/SyncMessageUtils.ts

```diff
@@ -0,0 +1,80 @@
+import {
+  ContactSyncMessage,
+  ContentMessage,
+  SyncMessageEnum,
+} from '../messages/outgoing';
+import { PubKey } from '../types';
+
+import * as _ from 'lodash';
+import * as Data from '../../../js/modules/data';
+import { ConversationController, textsecure, Whisper } from '../../window';
+import { OpenGroup } from '../types/OpenGroup';
+
+
+export async function from(
+  message: ContentMessage,
+  sendTo: PubKey | OpenGroup,
```

Comment:
`OpenGroup` shouldn't be part of this. Also wondering in what case would we need the pubkey?